DER/DerivativesContracts/Options.rdf

```diff
@@ -594,23 +469,40 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">An exercise schedule may be as simple as a single date or date period. However, in more complex cases, it may be an ad hoc schedule of individual dates, or a regular schedule of periodic exercise dates.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-opt;hasExerciseStyle">
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-opt;hasExerciseStyle">
 		<rdfs:subPropertyOf rdf:resource="&lcc-cr;isClassifiedBy"/>
 		<rdfs:label>has exercise style</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Option"/>
 		<rdfs:range rdf:resource="&fibo-sec-dbt-ex;ExerciseConvention"/>
 		<skos:definition>indicates the exercise convention specified for the option</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-opt;hasRelativePaymentDate">
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-opt;hasOptionHolder">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;hasContractParty"/>
+		<rdfs:label>has option holder</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Option"/>
+		<rdfs:range rdf:resource="&fibo-der-drc-opt;OptionHolder"/>
+		<skos:definition>indicates the owner of the option</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-opt;hasOptionWriter">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;hasContractParty"/>
+		<rdfs:label>has option writer</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Option"/>
+		<rdfs:range rdf:resource="&fibo-der-drc-opt;OptionIssuer"/>
+		<skos:definition>indicates the original issuer of the option</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Typically, the option writer collects the premium when the option is initially sold.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-opt;hasRelativePaymentDate">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDate"/>
 		<rdfs:label xml:lang="en">has relative payment date</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;RelativeDate"/>
 		<skos:definition xml:lang="en">The payment date for the Option Premium.</skos:definition>
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">This can be expressed as either an adjustable or relative date. FpML: &apos;The payment date, which can be expressed as either an adjustable or relative date.&apos;</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
this repeats the same phrase. And doesn't help explain it - see earlier comment

Reply:
@rivettp - hmmm ... not in ISO 704 format - likely the original MB definition and note.  Let's talk about this on Friday and fix it together.